src/app/plugins/colorpicker.plugin.ts

```diff
@@ -0,0 +1,30 @@
+import * as Canvas from 'canvas';
+import { MessageAttachment } from 'discord.js';
+import { Plugin } from '../../common/plugin';
+import { IContainer, IMessage, ChannelType } from '../../common/types';
+
+export class ColorPickerPlugin extends Plugin {
+  public name: string = 'Color Picker';
+  public description: string = 'Sends an image of the desired color';
+  public usage: string = 'colorpicker #<hex color>';
+  public pluginAlias = ['cp'];
+  public permission: ChannelType = ChannelType.Public;
+  public commandPattern: RegExp = /#[0-9a-f]{6}/;
+
+  constructor(public container: IContainer) {
+    super();
+  }
+
```

Comment:
why no validate method?

Reply:
I refactored it a little bit sometime ago
 The default [validate](https://github.com/cs-discord-at-ucf/lion/blob/6fff4915afaa7cbf9739b0688ab4ba3ef988ae95/src/common/plugin.ts#L29-L32) takes in a regex uses to test the args

Reply:
Regex is empty by default, so it returns true unless you give a public `commandPattern` or you can override it

Reply:
ahh was not aware. nice